cronback-proto/build.rs

```diff
@@ -16,6 +16,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 "./common.proto",
                 "./scheduler.proto",
                 "./dispatcher.proto",
+                "./project_srv.proto",
```

Comment:
Note to self: We should rename the rest of service protos for consistency. Maybe consider `_svc.proto` instead. _srv = server, _svc = service. The former represents the binary running with the role, the latter is the logical network service.